src/content/en/updates/2020/05/devtools.md

```diff
@@ -0,0 +1,126 @@
+
+Chromium Bug: [#1054381](https://crbug.com/1054381)
+
+<!--
+
+### Cumulative Layout Shift (CLS) movement records {: #cls }
+
+{# TODO(kayce): Get a working screenshot of the UI. }
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/f130314cc50ccd2c933017ee9bb06eefd68ed206 #}
+
+-->
+
+## Styles pane updates {: #styles }
+
+### Support for the `revert` keyword {: #revert }
+
+[revert]: https://developer.mozilla.org/en-US/docs/Web/CSS/revert
+[revert-bcd]: https://developer.mozilla.org/en-US/docs/Web/CSS/revert#Browser_compatibility
+
+The Styles pane's autocomplete UI now detects the [`revert`][revert] CSS keyword, which
+reverts the cascaded value of a property to what the value would have been if no changes
+had been made to the element's styling.
+
+![Setting the value of a property to `revert`](/web/updates/images/2020/05/revert.jpg)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/811c6e98ee25baf3f6f5a6145e1896fbe4362d7a #}
+
+Chromium Bug: [#1075437](https://crbug.com/1075437)
+
+### Image previews {: #image-previews }
+
+Hover over a `background-image` value in the Styles pane to see a preview of
+the image in a tooltip.
+
+![Hovering over a `background-image` value](/web/updates/images/2020/05/image-preview.jpg)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/1ebbb30be21f07d6a4d3ccd3b8dc7bd0dd37b41d #}
+
+Chromium Bug: [#1040019](https://crbug.com/1040019)
+
+### Support for space-separated functional color notations {: #functional-color-notation }
+
+[CSS Color Module Level 4](https://www.w3.org/TR/css-color-4/#changes-from-3)
+specifies that color functions like `rgb()` should support space-separated
+arguments. For example, `rgb(0, 0, 0)` is equivalent to `rbg(0 0 0)`. The Styles
+pane now recognizes the space-separated syntax.
+
+![Using space-separated arguments in the Styles pane](/web/updates/images/2020/05/color.jpg)
+
+The space-separated argument syntax is supported in most browsers. See the
+**Space-separated functional color notations** entry in MDN's
+[Browser compatibility](https://developer.mozilla.org/en-US/docs/Web/CSS/color#Browser_compatibility)
+table.
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/e858e4aa92e7b4f31de0d4eaccadba512bdc3c36 #}
+
+Chromium Bug: [#1072952](https://crbug.com/1072952)
+
+## Deprecation of the **Properties** pane {: #properties }
+
+The **Properties** pane in the **Elements** panel has been deprecated.
+Run `console.dir($0)` in the **Console** instead.
+
+![The Properties pane](/web/updates/images/2020/05/properties.jpg)
+
+References:
+
+* [`console.dir()`](/web/tools/chrome-devtools/console/api#dir)
+* [`$0`](/web/tools/chrome-devtools/console/utilities#dom)
+
+{# https://chromium.googlesource.com/devtools/devtools-frontend/+/5f6928b9570ed35c4a0b4c1de9b0d1f62370c636 #}
+
+<<../../_shared/devtools-feedback.md>>
```

Comment:
Some suggestions about a few changes that might be worth mentioning:

1) added basic a11y info in the inspector overlay https://chromiumdash.appspot.com/commit/380171a4329fdf8377c1454635e8495b02750442
2) added image preview in the styles panel https://chromiumdash.appspot.com/commit/1ebbb30be21f07d6a4d3ccd3b8dc7bd0dd37b41d
3) colors in the inspector overlay are formatted according to DevTools preferences (except for the original / as authored format) https://chromiumdash.appspot.com/commit/0848c278482016b16ae491bc08e2b937de7df18d

Reply:
I think 2 is already mentioned in the "Image previews" section.

1 is a huge feature, and I agree we should mention it: https://twitter.com/ChromeDevTools/status/1252275221777788929

3 could possibly be merged into the section about the new color formatting: https://github.com/google/WebFundamentals/pull/8700/files#r423487348

Reply:
Oh, number 1 is huge! There's been so much work on the accessibility of DevTools itself (which we do need to eventually announce) that I didn't realize it was a user-facing feature.

Reply:
I'm unclear about number 3 but we can follow-up about that in a subsequent PR